src/components/file-system-explorer/info/index.tsx

```diff
@@ -28,7 +28,7 @@ const FileExplorerInfo: FC<FileExplorerInfoProps> = ({
 }: FileExplorerInfoProps) => {
   const dateFormat = (date: string) => {
     // eslint-disable-next-line
-    return format(new Date(date), 'YYYY-MM-DD - HH:mm:ss');
+    return format(new Date(date), 'dd-MM-yyyy - HH:mm:ss');
```

Comment:
`yyyy-MM-dd  HH:mm:ss` is the format we use everywhere in the new UI, why did you update it?

It worth defining this format as a global variable as we use it very often

Reply:
Because the UI was broken. `YYYY-MM-DD` is not a valid format. Also it's not true that we use `yyyy-MM-dd` everywhere, see the screenshots.
![Screenshot from 2021-06-21 21-27-32](https://user-images.githubusercontent.com/3259802/122817002-b156f600-d2d7-11eb-94d0-7fcb5b947e23.png)
![Screenshot from 2021-06-21 21-27-14](https://user-images.githubusercontent.com/3259802/122817007-b1ef8c80-d2d7-11eb-9931-5345d94298fe.png)